pkg/components/render_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"testing"
 	"text/template"
 
+	embedded "github.com/openshift/microshift/assets"
```

Comment:
I would rather see:
1. remove the bindata.go
2. rename the package in `microshift/assets/embed.go` back to `assets` and change this wherever needed
3. profit :slightly_smiling_face:

Reply:
Problem is we already have `pkg/assets` package which imports `assets/`

Reply:
I see... I think I'll try to fix this afterwards, lets' stick with what you've got at this moment, then :slightly_smiling_face: